src/Data/Fin/Properties.agda

```diff
@@ -675,6 +675,23 @@ punchOut-punchIn (suc i) {suc j} = cong suc (begin
   j                                                           ∎)
   where open ≡-Reasoning
 
+
+------------------------------------------------------------------------
+-- pinch
+------------------------------------------------------------------------
+
+pinch-surjective : ∀ {m} i (j : Fin m) → Σ[ k ∈ Fin (suc m) ] (pinch i k ≡ j)
```

Comment:
Can we use the actual `Surjective` type from `Function`? Helps ensure consistency and therefore compatibility between different parts of the library!